pkg/manifest/init.go

```diff
@@ -185,8 +183,8 @@ func CreateDockerSecret(dockerConfigJSONFilename, ns string) (*ssv1alpha1.Sealed
 	if err != nil {
 		return nil, fmt.Errorf("failed to generate path to file: %w", err)
 	}
-
-	f, err := os.Open(authJSONPath)
+	fs := afero.NewOsFs()
```

Comment:
Can we pass in the fs (of other mechanism) instead of NewOsFs() again to reduce the number of times to construct it?   The benefit is that we would have less places to make changes to switch fs again.   Also, it sets up a better pattern for new codes.